kolibri/deployment/default/urls.py

```diff
@@ -26,4 +26,6 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^$', include('kolibri.core.urls')),
     url(r'', include('kolibri.content.urls')),
+    url(r'^api/', include('kolibri.auth.urls')),
```

Comment:
> What is the distinction between api and api-auth?

`api` is where our actual resources are mounted; `api-auth` is the endpoint for supporting logins from within the DRF browsable api (it could be named something else, but it's not itself a resource, so I don't think it would go under `api`):
http://www.django-rest-framework.org/tutorial/4-authentication-and-permissions/#adding-login-to-the-browsable-api